app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive: list[Animal] = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        if self.health > 0:
+            Animal.alive.append(self)
+
+    def die(self) -> None:
+        if self.health <= 0:
+            if self in Animal.alive:
+                Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
```

Comment:
The `__repr__` method should be updated to include the specific format for printing `Animal.alive` as required by the task. Consider adding logic to format the list of alive animals according to the task requirements.